docs/integrations/monitor-non-service-metrics-prometheus.rst

```diff
@@ -0,0 +1,101 @@
+Monitor non-service-specific metrics using Prometheus
+========================================================
+
+Prometheus integration allows you to monitor your Aiven services and understand the resource usage. Additionally, using this integration, you can monitor/track some non-service-specific metrics that may be worth monitoring. 
+
+To start monitoring these metrics, you need the :doc:`Prometheus integration configured and set up the Prometheus server <../platform/howto/integrations/prometheus-metrics>`.
+
+The complete list of metric you can monitor are not listed here. However, you can get the complete list in the Prometheus service or by sending ``HTTP GET`` with ``cURL`` as shown below: 
+
+.. code:: 
+  
+    curl --insecure --user promXXXX:XXXXXXXXXXXXXXXX https://YOUR-SERVICE-FQDN:9273/metrics
+
+
+CPU usage 
+----------
+CPU usage metrics are helpful in determining if the CPU is constantly being maxed out.
+For a high-level view of CPU usage for a single CPU service, you can use
+
+.. code:: 
+  
+    100 - cpu_usage_idle{cpu="cpu-total"}
+
+Bear in mind that it may not be a good idea to use ``cpu_usage_system{cpu="cpu-total"}`` + ``cpu_usage_user{cpu="cpu-total"}`` to calculate total CPU usage, because they do not include ``cpu_usage_iowait``. Furthermore, a process with a ``nice`` value larger than 0 will be categorized as ``cpu_usage_nice``, which is not included in ``cpu_usage_user``.
```

Comment:
I am not sure we need this part where we show someone something that "may not be a good idea". A simple explanation of the `nice` values could be useful, but it's kind of detailed for this, we could equally link to the upstream project docs on this where the `nice` values show up in the table below.

Reply:
I tried looking for more details in the current docs to link `nice` values to more details. Could you share the link to the upstream project where `nice` values are included?

Reply:
"nice" is a measure of how "nice" the process is to allow others to go ahead of it ... The citations seem to be really technical though, like https://man7.org/linux/man-pages/man1/nice.1.html

Reply:
I reduced the instruction a bit; I am not sure Aiven should be giving advice on some of these topics, but I've left the main points to get users started.